CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -1,3 +1,9 @@
+# [v1.8.5](https://github.com/kubermatic/kubeone/releases/tag/v1.8.5) - 2024-09-23
+
+## Changelog since v1.8.4
+
+**Note: the v1.8.4 release has been abandoned due to an issue with goreleaser deprecated flags.**
+
 # [v1.8.4](https://github.com/kubermatic/kubeone/releases/tag/v1.8.4) - 2024-12-23
```

Comment:
I'd drop the v1.8.4 heading because it doesn't lead to a working/usable release, let's have just v1.8.5 and leave a note about v1.8.4.

```suggestion
# [v1.8.5](https://github.com/kubermatic/kubeone/releases/tag/v1.8.5) - 2024-09-23

## Changelog since v1.8.4

**Note: the v1.8.4 release has been abandoned due to an issue with the [deprecated `goreleaser` flags](https://github.com/kubermatic/kubeone/pull/3519).**
```